docker/start_services.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Open these in the background
+memcached -d -U 0 -l 0.0.0.0 -m 64 -p 11211 &
+python /app/snappy/DiskCache.py -c /app/docker/dockerconfig.json &
+
+# Open this in the foreground so the docker container continues
+# to run
+python /app/snappy/SymServer.py -c /app/docker/dockerconfig.json
```

Comment:
I'm not super happy with this solution because it doesn't seem very flexible (ex: No easy way to increase the memory memcached uses). If all we need is an option for quickstart run SymServer in the foreground, I could add an option for that.